autoendpoint/Cargo.toml

```diff
@@ -45,14 +45,8 @@ tokio.workspace = true
 url.workspace = true
 uuid.workspace = true
 
-# Using a fork temporarily until these three PRs are merged:
-# - https://github.com/pimeys/a2/pull/49
-# - https://github.com/pimeys/a2/pull/48
-# - https://github.com/pimeys/a2/pull/47
-# The `autoendpoint` branch merges these three PRs together.
-# The version of a2 at the time of the fork is v0.5.3.
-a2 = { git = "https://github.com/mozilla-services/a2.git", branch = "autoendpoint" }
-# several of these libraries are pinned due to https://github.com/mozilla-services/autopush-rs/issues/249
+a2 = {version = "0.8", git = "https://github.com/mozilla-services/a2"}
```

Comment:
We should probably pin this to a specific revision (even though we own this repo) -- actually, upstream quickly merged your pr 🎉 so let's pin to their repo instead.

Reply:
?? https://github.com/WalletConnect/a2/pull/72 is still pending review.

I'll pin to a specific commit, but will still keep the version, since I ought to do a release.

Reply:
My fault I must have been looking at the wrong repo